src/pages/Registration/Steps/Documentation/Form/index.tsx

```diff
@@ -160,34 +159,15 @@ export default function Form() {
       </div>
 
       <Separator classes="my-8" />
-      {/*<CashEligibleCheckbox />*/}
-      <CheckField<FV>
-        name="hasAuthority"
-        required
-        classes={{
-          container: "check-field-reg text-sm mb-3",
-          input: "checkbox-reg self-start sm:self-center",
-          error: "mt-t",
-        }}
-      >
-        By checking this box, you declare that you have the authority to create
-        an endowment in the name of {data.contact.orgName} through {APP_NAME}
-      </CheckField>
-      <CheckField<FV>
-        name="hasAgreedToTerms"
-        required
-        classes={{
-          container: "check-field-reg text-sm",
-          input: "self-start sm:self-center",
-          error: "mt-1",
-        }}
-      >
-        By checking this box, you declare that you have read and agreed to our{" "}
-        <ExtLink className="underline text-orange" href={TERMS_OF_USE}>
+
+      <p className="text-sm">
+        By submitting this information, you declare that you have read and
+        agreed to our{" "}
+        <ExtLink className="underline text-orange" href={TERMS_OF_USE_NPO}>
```

Comment:
Not required for this PR, but we could add some `hover:` effect to this and related links